src/dune_rules/env_stanza_db.mli

```diff
@@ -0,0 +1,9 @@
+open Import
+
+val value
```

Comment:
Doesn't look like this is used anywhere.

Reply:
It's there for other parts of the code to use and implement their env like components without shoving it all in Env_node